src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -1218,11 +1194,8 @@ static void window_footpath_set_enabled_and_pressed_widgets()
         direction = gFootpathConstructValidDirections;
         if (direction != 255)
         {
-            disabledWidgets |=
-                (1 << WIDX_DIRECTION_NW) |
-                (1 << WIDX_DIRECTION_NE) |
-                (1 << WIDX_DIRECTION_SW) |
-                (1 << WIDX_DIRECTION_SE);
+            disabledWidgets
+                |= (1 << WIDX_DIRECTION_NW) | (1 << WIDX_DIRECTION_NE) | (1 << WIDX_DIRECTION_SW) | (1 << WIDX_DIRECTION_SE);
```

Comment:
Having the `|=` at the beginning of a line looks really odd to me

Reply:
Same as above, hopefully with different values for the penalties this can be avoided.